gcsfs/core.py

```diff
@@ -440,6 +442,11 @@ def _call(self, method, path, *args, **kwargs):
         data = kwargs.pop("data", None)
         r = None
 
+        if headers is None:
+            headers = {}
+        if "User-Agent" not in headers:
+            headers["User-Agent"] = "dask-gcsfs/" + VERSION
```

Comment:
I suggest `"python-gcsfs/"`, since this is not necessarily used in conjunction with Dask.